lib/util/http/bitbucket.spec.ts

```diff
@@ -3,6 +3,7 @@ import * as hostRules from '../host-rules';
 import { BitbucketHttp, setBaseUrl } from './bitbucket';
 
 const baseUrl = 'https://api.bitbucket.org';
+const range = (count: number) => [...Array(count).keys()];
```

Comment:
there is a range util function in test/utils

Reply:
This one? 
https://github.com/renovatebot/renovate/blob/4fe297233730ebcf7f81cf485521577e5df39c8f/lib/util/range.ts#L1-L9